include/ofi.h

```diff
@@ -374,6 +374,8 @@ static inline uint32_t ofi_xorshift_random_r(uint32_t *seed)
 
 uint32_t ofi_generate_seed(void);
 
+int ofi_vrb_check_fork_enabled(struct fid_domain *domain_fid);
```

Comment:
I feel the verb ```check``` should be moved up, like```ofi_check_vrb_fork_enabled```. Also ```vrb``` is not clear to me, may be 'ibv'?

Reply:
I was following the other verbs specific function in common -- `ofi_vrb_speed`.

Reply:
I see, thanks!